### ISSUE TITLE
[Popup] Popups don’t work with anchor tags when jQuery 3.2.0 is being used

### ISSUE THREAD

**Issue by iROCKBUNNY** [Semantic-Org/Semantic-UI#5186]
Here is the case:
```HTML
<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width">
  <title>JS Bin</title>
  <link rel="stylesheet" type="text/css" href="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.9/semantic.min.css">
</head>
<body>
  <a id="test" data-content="test" data-position="top left" data-variation="inverted">Test</a>
  <div id="test-div" data-content="test" data-position="top left" data-variation="inverted">Test div</div>
  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/jquery/3.2.0/jquery.min.js"></script>
  <script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.9/semantic.min.js"></script>
  <script>
    $('#test').popup(); // it does not work
    $('#test-div').popup(); // it works
  </script>
</body>
</html>
```

**Comment by Banandrew** [comment]
Hi @iROCKBUNNY, I’ve just tried it locally, and it works as expected. Do you get any errors in the console?

**Comment by iROCKBUNNY** [comment]
Hi @Banandrew , thanks for your response!

I pasted the code on jsbin (sorry I don't know how to create a jsbin link), and the first case doesn't work. 

In my case, I am using Chrome 56 on macOS 10.12.3.

By the way, if I change jQuery to v3.1.1, both case works. (same on jsbin)

**Comment by Banandrew** [comment]
@iROCKBUNNY but do you get any console errors on JSBin? It seems that when pulled from a CDN (any CDN, I tried two), 3.2 produces an `a.indexOf is not a function` error at line 9814 when unminified, can you please confirm?

**Comment by iROCKBUNNY** [comment]
@Banandrew I just tried my code locally and on jsbin with unminified js files, after reading your message. I did not find an error message in the console (maybe I did not do it correctly). However, I can see the rendering of the popup html is not functioning with `<a>` tag. It tries to create the following code when the `<a>` popup holder is triggered (hover the mouse on it): 
```html
<div class="ui popup inverted">
  <div class="content">test</div>
</div>
```
and does not destruct it when moving the mouse away. (unlike the `<div>` popup)

**Comment by Banandrew** [comment]
@iROCKBUNNY disregard my last message, it seems to be my lack of understanding of how JSFiddle works, and CDNs do actually work. I’ve just followed your setup, and reproduced the issue.

**Comment by iROCKBUNNY** [comment]
Hi @Banandrew , the issue is solved when jQuery v3.2.1 is being used.

**Comment by Banandrew** [comment]
Hi @iROCKBUNNY, thanks, can confirm. Closing the issue—it seems that the problem wasn’t on SUI’s end.
